lib/miq_automation_engine/service_models/miq_ae_service_generic_object.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServiceGenericObject < MiqAeServiceModelBase
+    def method_missing(method_name, *args)
+      @object.send(method_name, *args)
+    end
+
+    def respond_to_missing?(method_name, include_private = false)
```

Comment:
@gmcculloug I corrected `respond_to_missing?`. It is always paired with `method_missing`